services/core/shared/core/compat/sdk_v5/voters.js

```diff
@@ -137,10 +143,10 @@ const getVoters = async params => {
 	votes.data.account = {
 		address: params.receivedAddress,
 		username: accountInfo && accountInfo.username ? accountInfo.username : undefined,
-		votesUsed: votes.data.votes.length,
+		votesReceived: votes.data.votes.length,
```

Comment:
After including the support for `limit` and `offset`, `votesReceived` can be updated with `votesAggregateDB.count({.receivedAddress: params.receivedAddress })`.

Reply:
Done!